[BUG] AAD variables name conflict

### Community Note

The code has `entra-*` naming of [variables for VPN](https://github.com/claranet/terraform-azurerm-vpn/blob/master/variables-vpn.tf#L151-L153) in `vpn_client_configuration` block, but tries to use variables with `aad-*` naming in [VPN resource code](https://github.com/claranet/terraform-azurerm-vpn/blob/master/r-vpn.tf#L74-L76).

This makes module undeployable:

```
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 74, in resource "azurerm_virtual_network_gateway" "main":
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_issuer".
╵
##[error]Terraform command 'plan' failed with exit code '1'.
##[error]╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 74, in resource "azurerm_virtual_network_gateway" "main":
│   74:       aad_tenant    = vpn.value.aad_tenant
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_tenant".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 75, in resource "azurerm_virtual_network_gateway" "main":
│   75:       aad_audience  = vpn.value.aad_audience
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_audience".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 76, in resource "azurerm_virtual_network_gateway" "main":
│   76:       aad_issuer    = vpn.value.aad_issuer
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_issuer".
╵

```

<!--- Please keep this note for the community --->

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->


### OpenTofu Version

0.0.0

### AzureRM Provider Version

4

### Affected Resource(s)/Data Source(s)

azurerm_virtual_network_gateway

### OpenTofu Configuration Files

```hcl
Using Terraform
```

### Debug Output/Panic Output

```shell
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 74, in resource "azurerm_virtual_network_gateway" "main":
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_issuer".
╵
##[error]Terraform command 'plan' failed with exit code '1'.
##[error]╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 74, in resource "azurerm_virtual_network_gateway" "main":
│   74:       aad_tenant    = vpn.value.aad_tenant
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_tenant".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 75, in resource "azurerm_virtual_network_gateway" "main":
│   75:       aad_audience  = vpn.value.aad_audience
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_audience".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/***/r-vpn.tf line 76, in resource "azurerm_virtual_network_gateway" "main":
│   76:       aad_issuer    = vpn.value.aad_issuer
│     ├────────────────
│     │ vpn.value is object with 10 attributes
│ 
│ This object does not have an attribute named "aad_issuer".
╵
```

### Expected Behaviour

_No response_

### Actual Behaviour

_No response_

### Steps to Reproduce

_No response_

### Important Factoids

_No response_

### References

_No response_

--- a/r-vpn.tf
+++ b/r-vpn.tf
@@ -71,9 +71,9 @@ resource "azurerm_virtual_network_gateway" "main" {
 
     content {
       address_space = vpn.value.address_space
-      aad_tenant    = vpn.value.aad_tenant
-      aad_audience  = vpn.value.aad_audience
-      aad_issuer    = vpn.value.aad_issuer
+      aad_tenant    = vpn.value.entra_tenant
+      aad_audience  = vpn.value.entra_audience
+      aad_issuer    = vpn.value.entra_issuer
 
       dynamic "root_certificate" {
         for_each = vpn.value.root_certificate

